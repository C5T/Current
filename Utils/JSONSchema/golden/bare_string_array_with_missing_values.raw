{"Array":{"element":{"String":{"values":{"thing":1,"this":1,"works":1},"instances":3,"nulls":2}},"instances":1,"nulls":0}}