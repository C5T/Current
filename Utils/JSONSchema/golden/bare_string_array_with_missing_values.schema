// Autogenerated schema inferred from input JSON data.

using Schema = std::vector<Optional<std::string>>;
