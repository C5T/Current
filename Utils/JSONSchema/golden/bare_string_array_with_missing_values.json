["this",null,"thing",null,"works"]
